### ISSUE TITLE
Exclude users for which Member Service cannot return details and try again

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#3768]
When we are loading topics with posts, we are also loading user details so we can show user avatars, full name, handle and so on for topics and posts:

![image](https://user-images.githubusercontent.com/146016/76400655-9c44a500-63bb-11ea-8d8e-de8485a1f692.png)

We do it here https://github.com/appirio-tech/connect-app/blob/hotfix/show-topics-without-user-details/src/projects/actions/projectTopics.js#L58-L74

For some users, Member Service cannot return details (for example delated users) so when we request data about several users the whole request may fail because of one user and we wouldn't see information even about users for which we can get data.

We should implement functionality with the next logic:
- we request details about all the users at once (as we do now)
- if the requests fails, we have to remove the user for which request fails and try to get data for users again
- if the request fails again, we should exclude the next user which causes it to fail
- and we should do this until we get data for the users, for which Member Service can return data properly
- this should be implemented inside action `loadMembers` https://github.com/appirio-tech/connect-app/blob/dev/src/actions/members.js#L5-L22.
   - add the second argument to this action `skipErrors`. If it's set to `true` then we should exclude users with errors as described in this issue. If it's `false` then we should request users only once, and if it fails, then it fails.

**Comment by maxceem** [comment]
@vikasrohit I've tried to request data for some non-existent users with userId `1` or `123`. The member service doesn't return data for them, but it doesn't fail. Do you have any example userId for which Member Service would fail?

**Comment by vikasrohit** [comment]
I can do that for some user, do you have a user which we can use for this purpose?

**Comment by maxceem** [comment]
Let's do this to one of my users `maxceem12` with userId `40159058`.

**Comment by vikasrohit** [comment]
Ok, please create a post in sample project using this user before I deactivate this user.

**Comment by maxceem** [comment]
Created in this project https://connect.topcoder-dev.com/projects/8912

**Comment by vikasrohit** [comment]
Done. User is inactive now and we should see the error in _search api call now.

**Comment by maxceem** [comment]
Looks like there would be a tricky moment. We request several users by their `userId` like this:
https://api.topcoder-dev.com/v3/members/_search/?fields=userId%2Chandle%2CphotoURL%2CfirstName%2ClastName%2Cdetails%2Cemail&query=userId%3A40152856%20OR%20userId%3A40159058&limit=2

and when it fails for some user it returns error with `handle` instead of `userId`:

![image](https://user-images.githubusercontent.com/146016/76412182-24cc4100-63ce-11ea-89f8-f9c3ff69fea2.png)

So we don't know which users causes the error.

**Comment by vikasrohit** [comment]
Closing the issue as won't fix because member api is fixed to not throw error when one of the user is inactive in passed query string.
